hikari/internal/enums.py

```diff
@@ -122,8 +109,15 @@ def __call__(cls, value: typing.Any) -> typing.Any:
     def __getitem__(cls, name: str) -> typing.Any:
         return cls._name_to_member_map_[name]
 
+    def __contains__(cls, item: typing.Any) -> bool:
+        try:
+            cls._value_to_member_map_[item]
+            return True
+        except KeyError:
+            return False
+
     def __iter__(cls) -> typing.Iterator[str]:
```

Comment:
would iterating values here not make it an iterator of Any not str

Reply:
Yep, will fix typehint